app/controllers/to_do_items_controller.rb

```diff
@@ -4,7 +4,7 @@ def new
     list = List.find params[:id]
     @item = list.items.new
 
-    render partial: 'to_do_items/form', locals: {item: @item}
+    render partial: 'new'
```

Comment:
@vlasiak, What if we make a partial as a view (simply rename `_new` into `new`)? It looks we would not need even call render here, right?